evm/multisig.js

```diff
@@ -0,0 +1,325 @@
+    }
+
+    const threshold = await multisigContract.signerThreshold();
+
+    if (voteCount.eq(threshold.sub(1)) && !yes) {
+        printWarn(`The vote count is one below the threshold. This action will execute the multisig proposal.`);
+        const answer = readlineSync.question(`Proceed with ${multisigAction}?`);
+        if (answer !== 'y') return 0;
+    }
+}
+
+async function processCommand(options, chain) {
+    const {
+        contractName,
+        address,
+        multisigAction,
+        symbols,
+        limits,
+        mintLimiterAddress,
+        recipientAddress,
+        withdrawAmount,
+        target,
+        calldata,
+        nativeValue,
+        privateKey,
+        yes,
+    } = options;
+
+    const contracts = chain.contracts;
+    const contractConfig = contracts[contractName];
+
+    let multisigAddress;
+
+    if (isAddress(address) && address !== AddressZero) {
+        multisigAddress = address;
+    } else {
+        if (!contractConfig?.address) {
+            throw new Error(`Contract ${contractName} is not deployed on ${chain.name}`);
+        }
+
+        multisigAddress = contractConfig.address;
+    }
+
+    const rpc = chain.rpc;
+    const provider = getDefaultProvider(rpc);
+
+    const wallet = new Wallet(privateKey, provider);
+    await printWalletInfo(wallet);
+
+    printInfo('Contract name', contractName);
+
+    const multisigContract = new Contract(multisigAddress, IMultisig.abi, wallet);
+
+    const gasOptions =  contractConfig?.gasOptions || chain?.gasOptions || {};
+    console.log(`Gas override for chain ${chain.name}: ${JSON.stringify(gasOptions)}`);
+
+    printInfo('Multisig Action', multisigAction);
+
+    switch (multisigAction) {
+        case 'setTokenMintLimits': {
+            const symbolsArray = JSON.parse(symbols);
+            const limitsArray = JSON.parse(limits);
+
+            if (!isStringArray(symbolsArray)) {
+                throw new Error(`Invalid token symbols: ${symbols})}`);
+            }
+
+            if (!isNumberArray(limitsArray)) {
+                throw new Error(`Invalid token limits: ${limits}`);
+            }
+
+            if (symbolsArray.length !== limitsArray.length) {
+                throw new Error('Token symbols and token limits length mismatch');
+            }
+
+            const multisigTarget = chain.contracts.AxelarGateway?.address;
+
+            if (!isAddress(multisigTarget) && multisigTarget !== AddressZero) {
+                throw new Error(`Missing AxelarGateway address in the chain info.`);
+            }
+
+            const gatewayContract = new Contract(multisigTarget, IGateway.abi, wallet);
+            const targetInterface = new Interface(gatewayContract.interface.fragments);
+            const multisigCalldata = targetInterface.encodeFunctionData('setTokenMintLimits', [symbolsArray, limitsArray]);
+
+            await preExecutionChecks(multisigContract, multisigAction, wallet, multisigTarget, multisigCalldata, 0, yes);
```

Comment:
`yes` doesn't need to be passed in. we can just ask a question here right before execution to proceed